server/routes/phase-allocation.js

```diff
@@ -39,6 +44,26 @@ router.get(
   })
 );
 
+// Read: Get phases globally
```

Comment:
Assuming you mean not in ticket, it's not really feasible to implement a page for something that doesn't have an endpoint to reach. But yes, ideally this should have been multiple tickets.

Reply:
Oh sorry, no i meant the comment to describe what this route does should say `Get global phases` rather than `Get phases globally`. Semantically makes more sense, as the thing being requested is a "global phase"